### PR TITLE
docs: add config:elemental:registration section

### PR DESCRIPTION
**Pull request by fgiudici**
update also the machineName to the new default

### CODE REVIEW

File: docs/machineregistration-reference.md [hunk 1]
```diff
@@ -75,9 +91,10 @@ This refers to the name that will be set to the node and the kubernetes resource
 `String` type.
 
 !!! info
-    machineName is interpolated with [SMBIOS](https://www.dmtf.org/standards/smbios) data which allows you to store hardware
-    information.
-    See our [SMBIOS docs](smbios.md) for more information
+    when `elemental:registration:no-smbios` is set to `false` (default) machineName is interpolated with [SMBIOS](https://www.dmtf.org/standards/smbios) data which allows you to store hardware information.
```
Comment (Itxaka) [inline docs/machineregistration-reference.md:94]: can you add this line below also ot have consistent notes about smbios (line 116)?

**Review by Itxaka — APPROVED** [review]
only one nit

File: docs/machineregistration-reference.md (hunk 1 above)
Comment (fgiudici) [reply docs/machineregistration-reference.md]: yep, good catch, I missed that!

**Review by Itxaka — APPROVED** [review]

**Comment by Itxaka** [comment]
cc @nunix

File: docs/machineregistration-reference.md (hunk 1 above)
Comment (fgiudici) [reply docs/machineregistration-reference.md]: done

**Pull request merged**